client/src/components/AddFriend/AddFriend.jsx

```diff
@@ -31,22 +27,17 @@ class AddFriends extends Component {
 		return null;
 	}
 
-	componentDidMount = () => {
-		console.log('add friends forms is mounting');
-	};
-
 	handleChange = e => {
 		const { name, value } = e.target;
 		this.setState({ [name]: value });
 	};
 
 	handleSubmit = e => {
-		e.preventDefault();
+		if (e) e.preventDefault();
```

Comment:
Is there a situation where we are not getting an event as an arg?

Reply:
Just when we are testing and calling the prop directly. It's something we can mock in the test, but I thought it is just a little tidier putting this here.